bin/wmagent-mod-config

```diff
@@ -150,6 +150,9 @@ def modifyConfiguration(config, **args):
     if hasattr(config, "DBS3Upload"):
         config.DBS3Upload.dbsUrl = args["dbs3_url"]
 
+    if hasattr(config, "WorkflowUpdater"):
+        config.WorkflowUpdater.dbsUrl = args["dbs3_reader_url"]
```

Comment:
I would suggest grouping this together with the other parameters for WorkflowUpdater, see https://github.com/dmwm/WMCore/blob/master/bin/wmagent-mod-config#L292 (btw, please fix the comment line above, which has the wrong component name)

Reply:
In addition, not to break T0 deployment, I would suggest making it backwards compatible (even if that means we can potentially start an agent with the default "OVER_WRITE_BY_SECRETS" value.

Reply:
Added None as default value

Reply:
I think you forgot to group the WorkflowUpdater attributes together.